README.md

```diff
@@ -166,6 +166,44 @@ The following `customize` subkeys are available. They inherit from the specified
  - `add_disks` - Array of disks to add to the VM (requires VirtualMachine.Config.AddNewDisk).
    Keys per disk: `type` (default: `thin`, other values: `flat`/`flat_lazy` or `flat_eager`), `size_mb` in MB (default: 10 GB)
 
+The following `guest_customization` subkeys are available. To configure the guest OS you must define all of the provided parameters. Below the parameters is an example of their usage in a platform driver configuration.
+
+ - `ipAddress` - String for configuring a static IPv4 address (performs validation as IPv4 only is supported at this time)
+ - `gateway` - Array for configuring IPv4 addresses as gateways
+ - `subnetMask` - String for configuring subnet mask
+ - `dnsDomain` - String for configuring DNS domain
+ - `timezone` - String for configuring timezone
+ - `dnsServerList` - Array for configuring DNS servers
+ - `dnsSuffixList` - Array for configuring DNS suffixes
+
+```yml
+platforms:
+  - name: centos-7
+    driver:
+      datacenter: 'Datacenter'
+      template: 'centos-7-template'
+      cluster: 'Kitchen'
+      interface: 'Kitchen Network'
+      poweron: true
+      vm_name: centos-7-kitchen
+      guest_customization:
+        ipAddress: 10.10.176.15
```

Comment:
Perhaps consider having the new attributes be in snake_case - like `ip_address` rather than `ipAddress` - for consistency with the rest of the driver

Reply:
@clintoncwolfe thank you for that suggestion and that makes absolute sense. I was torn about that as the VMware SDK uses that casing, but I like the consistency this brings.